packages/test/test-service-load/src/runner.ts

```diff
@@ -301,7 +299,10 @@ function scheduleContainerClose(
         // wait for the container to connect write
         container.once("closed", () => resolve);
         // TODO: Remove null check after next release #8523
-        if (container.connected !== undefined && !container.connected && !container.closed) {
+        if (container.connectionState !== undefined
```

Comment:
Why this undefined check?

Reply:
I included it to be consistent with the previous undefined check, but it seems unnecessary (and the comment above it implies this too).